content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,48 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The control plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside groups of one or more
+co-located containers called
+{{< glossary_tooltip text="_pods_" term_id="pod" >}}. Containers within a
+pod may share resources, such as an IP address and storage volumes, whereas
+pods are isolated from one another and from their hosts.  
```

Comment:
Here are some questions that came to mind:
Is there a reason to mention `hosts` and `nodes`?
Does Pod == colocated containers?

Reply:
Although the control plane is where the magic happens, readers new to Kubernetes need to understand what nodes do before they learn about the thing that controls those nodes.

Reply:
Not sure about this. I tend to look at the complete picture and focus on the API.